src/components/shared/card.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Modal, Button, Row, Col} from 'react-bootstrap';
+import Mask from '../../icons/gear_types/mask.png';
+import Backpack from '../../icons/gear_types/backpack.png';
+import Vest from '../../icons/gear_types/vest.png';
+import Glove from '../../icons/gear_types/gloves.png';
+import Holster from '../../icons/gear_types/holster.png';
+import Kneepads from '../../icons/gear_types/kneepads.png';
+import Assault from '../../icons/weapons/AssaultRifle.png';
+import LMG from '../../icons/weapons/Light Machine Gun.png';
```

Comment:
**Tech debt**: Consider renaming to avoid the syntax highlighting screwing up. Very minor but for clarity.